packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -81,6 +81,15 @@ ${postGetLink(post, true, false)}
   return `mailto:?subject=${encodeURIComponent(subject)}&body=${encodeURIComponent(body)}`;
 };
 
+// Select the social preview image for the post, using the manually-set
+// cloudinary image if available, or the auto-set from the post contents. If
+// neither of those are available, it will return null.
+export const getSocialPreviewImage = (post: DbPost): string => {
+  const manualId = post.socialPreviewImageId
+  if (manualId) return `https://res.cloudinary.com/cea/image/upload/c_fill,ar_1.91,g_auto/${manualId}`
```

Comment:
This will presumably break for us? Given the `cea` domain. Would be great to factor this out into a setting.